x-pack/plugins/fleet/common/constants/preconfiguration.ts

```diff
@@ -5,7 +5,57 @@
  * 2.0.
  */
 
+import { i18n } from '@kbn/i18n';
+
+import type { PreconfiguredAgentPolicy } from '../types';
+
+import { defaultPackages } from './epm';
+
 export const PRECONFIGURATION_DELETION_RECORD_SAVED_OBJECT_TYPE =
   'fleet-preconfiguration-deletion-record';
 
 export const PRECONFIGURATION_LATEST_KEYWORD = 'latest';
+
+export const DEFAULT_AGENT_POLICY: PreconfiguredAgentPolicy = {
+  name: 'Default policy',
+  namespace: 'default',
+  description: 'Default agent policy created by Kibana',
+  package_policies: [
+    {
+      name: i18n.translate('xpack.fleet.agentPolicies.defaultAgentPolicySystemIntegrationName', {
+        defaultMessage: 'System',
+      }),
+      package: {
+        name: defaultPackages.System,
+      },
+    },
+  ],
+  is_default: true,
+  is_managed: false,
+  monitoring_enabled: ['logs', 'metrics'] as Array<'logs' | 'metrics'>,
+  id: 'default-agent-policy',
```

Comment:
do we really need to specify ID here? this will set a `preconfiguration_id` on the SO, right? which will cause different behavior downstream

Reply:
The Preconfiguration API expects the `id` field when creating an agent policy, but with the `is_default`/`is_default_fleet_server` flags preventing the policies from being deleted, it won't really come into play after initial creation.

Reply:
understood. I would still like to avoid setting `preconfiguration_id` for the policies here though, I can still see a possibility of this field used in the future in other places and unintentionally affecting these system default policies too. is it a large effort to not set these IDs?

Reply:
Pushed a commit to make `preconfiguration_id` optional if either `is_default` or `is_default_fleet_server` are `true`. Otherwise the API will throw an error.